types/src/front/assistant/agent.ts

```diff
@@ -56,6 +56,7 @@ export type AgentGenerationConfigurationType = {
   id: ModelId;
   model: SupportedModel;
   temperature: number;
+  forceUseAtIteration: number | null;
```

Comment:
3 fields if we count id & sId. 
Since we're going to add many actions soon, good to have the type checker remind people to fill those 3 fields (e.g. I think sId is missing on GenerationConfiguration)

maybe outside scope of the PR but would you think it legitimate to have a separate PR (by me, you or someone else) for that?

Reply:
fair fair, happy to do that

Reply:
Let's do it as soon as we move generation to actions so we don't have to do weird imports

Reply:
seeing as we have 4 actions sharing 3 fields, and will soon have 3 more actions, isn't that making it easier for us to automate these 3 fields? VS mental load for people introducing new actions to add the fields with the right consistency everywhere needed? 

If we're sure we're going to add many actions, and will want to treat them partly similarly, a common ground seems to makes sense?

(just pulling the argument a bit, won't fight harder than that for this)

Reply:
Factorising actions is a much broader project and we’ll need to think hard about how we want to do it (likely using TS classes and inheritance to have all implementation of an action in the same place + generic code for interaction with it in our conversations logic)